docs/api-overview.mdx

```diff
@@ -181,11 +181,14 @@ An example Credentials object is below:
 ```typescript title="Credentials"
 {
   "type": string, // credential type
-  "data": Map<string, any> // credential data map that matches the credential type's JSON Schema definition
+  "data": Map<string, any>, // credential data map that matches the credential type's JSON Schema definition
+  "id"?: string, // credential id; optional. If not provided, a random uuid will be generated
   "expirationDate?": number // when the credential expires as a milliseconds since unix epoch; optional
 }
 ```
 
+While, not required, we recommend providing an `id` for each credential. This allows for idempotent credential issuance - meaning if a duplicate `id` value is seen another credential will not be created. Expected a https status code of `409`, if this is the case. If you do not provide an `id`, a random uuid will be generated for you.
```

Comment:
```suggestion
While not required, we recommend providing a unique `id` for each credential. This allows for idempotent credential issuance - meaning if a duplicate `id` value is seen another credential will not be created. Expect a https status code of `409` if this is the case. If you do not provide an `id`, a random uuid will be generated for you.
```

Reply:
I'm not sure how specific we need to be in our wording here. I definitely think we should at least say that the `id` needs to be unique. Maybe we should even enforce that it's a uuid?

Reply:
We actually are enforcing is a uuid. Good call calling out that requirement